src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -56,3 +57,9 @@ def get(self, svc_id):
             'osd_metadata': mgr.get_metadata('osd', svc_id),
             'histogram': histogram,
         }
+
+    @RESTController.Resource('POST', query_params=['deep'])
+    def scrub(self, svc_id, deep=False):
+        deep = deep.lower()
```

Comment:
Nah, this won't work. @p-na has a similar problem in his PR. There is a function to convert strings to Booleans in the pool controller. Might be worth investigating.

Reply:
I have extracted the method from the pool controller and I'm using it now.